R/derive_var_disposition_status.R

```diff
@@ -1,5 +1,9 @@
 #' Default Format for Disposition Status
 #'
+#' @description
+#' `r lifecycle::badge("deprecated")`
+#'
+#' This function is *deprecated*, please use `derive_var_merged_cat()` instead.
```

Comment:
I think you can not use `derive_var_merged_cat()` instead of `format_eoxxstt_default()` but need to define your own function.